recon-enumeration/index.md

```diff
@@ -0,0 +1,211 @@
+* **Battering ram** - This uses a single set of payloads.
+It iterates through the payloads, and places the same payload into all of the defined payload positions at once.
+This attack type is useful where an attack requires the same input to be inserted in multiple places within the request (e.g. a username within a Cookie and a body parameter).
+The total number of requests generated in the attack is the number of payloads in the payload set.
+* **Pitchfork** - This uses multiple payload sets.
+There is a different payload set for each defined position (up to a maximum of 20).
+The attack iterates through all payload sets simultaneously, and places one payload into each defined position.
+In other words, the first request will place the first payload from payload set 1 into position 1 and the first payload from payload set 2 into position 2; the second request will place the second payload from payload set 1 into position 1 and the second payload from payload set 2 into position 2, etc.
+This attack type is useful where an attack requires different but related input to be inserted in multiple places within the request (e.g. a username in one parameter, and a known ID number corresponding to that username in another parameter).
+The total number of requests generated in the attack is the number of payloads in the smallest payload set.
+* **Cluster bomb** - This uses multiple payload sets.
+There is a different payload set for each defined position (up to a maximum of 20).
+The attack iterates through each payload set in turn, so that all permutations of payload combinations are tested. I.e., if there are two payload positions, the attack will place the first payload from payload set 2 into position 2, and iterate through all the payloads in payload set 1 in position 1; it will then place the second payload from payload set 2 into position 2, and iterate through all the payloads in payload set 1 in position 1.
+This attack type is useful where an attack requires different and unrelated or unknown input to be inserted in multiple places within the request (e.g. when guessing credentials, a username in one parameter, and a password in another parameter).
+The total number of requests generated in the attack is the product of the number of payloads in all defined payload sets - this may be extremely large.
+
+There are also many different types of payloads you can use ([here](https://portswigger.net/burp/documentation/desktop/tools/intruder/payloads/types) is the entire list), from specifying your own list of words to generating random bytes. You can find lists of popular credentials online, for instance, [here](https://github.com/danielmiessler/SecLists/tree/master/Passwords) is a repo with lists of most used passwords.
+
+## Web Content Enumeration/ Web Content Discovery/ Dirbusting
+
+Let's say we have the target server **http://192.168.1.224/** and we want to discover hidden files, directories or other resources there.
```

Comment:
I think it's better to give a real example and replace that local IP.
Maybe something like `https://security.cs.pub.ro/`.
What do you think?